src/hooks/use-upload.tsx

```diff
@@ -37,6 +37,7 @@ const useUpload = ({ files, validationData, emitter }: UseUpload) => {
     const error = filesValidation(newFiles, validationData) ?? ''
     const filesForEmitter = error ? files : newFiles
     emitter({ files: filesForEmitter, error })
+    event.target.value = ''
```

Comment:
It is a bad practice to directly mutate function arguments. You should avoid this. I believe there should be a better solution for your problem

Reply:
Used ref istead. After button click we resetting previous data to solve this problem